src/vendor/Json.res

```diff
@@ -4,16 +4,16 @@ module Encode = Json_encode
 exception ParseError(string)
 
 let parse = s =>
-  try Some(JSON.parseExn(s)) catch {
+  try Some(JSON.parseOrThrow(s)) catch {
   | _ => None
   }
 
 let parseOrRaise = s =>
-  try JSON.parseExn(s) catch {
-  | Exn.Error(e) =>
-    let message = switch Exn.message(e) {
+  try JSON.parseOrThrow(s) catch {
+  | JsExn(e) =>
+    let message = switch JsExn.message(e) {
     | Some(m) => m
     | None => "Unknown error"
     }
-    \"@@"(raise, ParseError(message))
+    throw(ParseError(message))
```

Comment:
I'm actually wondering if we should keep this outdated Json encoder/decoder vendored lib, using plain rescript should be much better now that we even have pattern matching on dict with v12! But this could be part of another PR.

Reply:
def. stuff for another PR